.travis.yml

```diff
@@ -14,6 +14,6 @@ script:
 deploy:
   - provider: script
     skip_cleanup: true
-    script: workshop/deploy_rollup.sh
+    script: echo workshop/deploy_rollup.sh
```

Comment:
Why is it `echo` here?

Reply:
You can create one here https://console.bluemix.net/iam/#/apikeys (the account should be the one who has access to the `CLUSTER_NAME` and the namespace)

For the namespace,
https://github.com/IBM/kube101/blob/2633eed2a89b90c80387ec3001caba9820b85d2c/workshop/deploy.sh#L76

I'm guessing its `contbot`?
https://github.com/IBM/kube101/blob/2633eed2a89b90c80387ec3001caba9820b85d2c/workshop/deploy.sh#L49

Reply:
yea, I'm sure its possible, but do we have an existing cluster/org/space/.... setup for testing already?